Sources/key-cast/KeyCast.swift

```diff
@@ -87,32 +116,39 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
     }
 
     func updateText(_ untrimmed: String, onlyMeta: Bool = false) {
-        let str = untrimmed.trimmingCharacters(in: .whitespacesAndNewlines)
+        let string = untrimmed.trimmingCharacters(in: .whitespacesAndNewlines)
 
-        if str.count == 0 {
+        if string.count == 0 {
             if timer == nil {
                 window.orderOut(self)
             }
             return
         } else if onlyMeta {
-            self.timer?.invalidate()
-            self.timer = nil
+            timer?.invalidate()
+            timer = nil
         } else {
-            self.queueClear()
+            queueClear()
         }
 
         let windowFrame = window.frame
 
         var originalFrame = field.frame
         originalFrame.size.width = 500 // Max width of the view
         field.frame = originalFrame
-
-        field.textStorage?.setAttributedString(NSAttributedString(string: str))
-        field.font = NSFont(name:"Helvetica Bold", size: self.windowSize.fontSize)
+        field.textStorage?.setAttributedString(NSAttributedString(string: string))
         field.textColor = .textColor
-        field.alignment = .center        
+        field.alignment = .center
 
-        guard let layoutManager = field.layoutManager, let textContainer = field.textContainer else {
+        if #available(macOS 10.15, *) {
+           field.font = NSFont.monospacedSystemFont(ofSize: windowSize.fontSize, weight: .bold)
```

Comment:
This is the only functional change. I think it's better to use a monospace font for the output, for readability and it will makes it not move when using different keys.